### DISCUSSION TITLE
i want to run composer update command

### DISCUSSION THREAD

**Discussion by vahidalvandi** [spatie/laravel-backup#1672] [General]
hello
i want to run composer `update command` but return error , please show how i can run composer update  

this my code
```

    $process = new Process(['composer', 'u'],base_path(),[],null,300);
    $process->run();

    // executes after the command finishes
    if (!$process->isSuccessful()) {
        throw new ProcessFailedException($process);
    }

    echo $process->getOutput();
```

**The command "'composer' 'u'" failed. Exit Code: 1(General error) Working directory: /media/hh/storage/laravel/mg-v3 Output: ================ Error Output: ================ [RuntimeException] The HOME or COMPOSER_HOME environment variable must be set for composer to run correctly**

**Comment by parallels999** [comment]
That doesn't seem like a problem with this package.
